frontend/src/App.js

```diff
@@ -119,14 +110,57 @@ function App() {
             setIndustrySpeaker={setIndustrySpeaker}
           />
         </Route>
+
+        <Route path='/profile'> 
+          <ProfilePage 
+            loggedUser={loggedUser}
+            setFeedback={setFeedback} 
+            setUser={setUser}
+            email={email}
+            setEmail={setEmail}
+            password={password}
+            setPassword={setPassword}
+            firstName={firstName}
+            setFirstName={setFirstName}
+            lastName={lastName}
+            setLastName={setLastName}
+            newPassword={newPassword}
+            setNewPassword={setNewPassword}
+
+            cohortId={cohortId}
+            setCohortId={setCohortId}
+
+            company={company}
+            setCompany={setCompany}
+            title={title}
+            setTitle={setTitle}
+            volunteerSkills={volunteerSkills}
+            setVolunteerSkills={setVolunteerSkills}
+            mentor={mentor}
+            setMentor={setMentor}
+            officeHours={officeHours}
+            setOfficeHours={setOfficeHours}
+            techMockInterview={techMockInterview}
+            setTechMockInterview={setTechMockInterview}
+            behavioralMockInterview={behavioralMockInterview}
+            setBehavioralMockInterview={setBehavioralMockInterview}
+            professionalSkillsCoach={professionalSkillsCoach}
+            setProfessionalSkillsCoach={setProfessionalSkillsCoach}
+            hostSiteVisit={hostSiteVisit}
+            setHostSiteVisit={setHostSiteVisit}
+            industrySpeaker={industrySpeaker}
+            setIndustrySpeaker={setIndustrySpeaker}
+          />
```

Comment:
i have a feeling a custom hook to set all the state variables wouldve saved you some work here by only needing to pass one prop for all the sets instead of 1:1 but good work nonetheless

Reply:
agree on that, just I still don't know how to do it, I can look at it later at a different stage and remanufacture it